backend/tests/Service/Management/Health/AllActiveIpsHaveCorrectPtrHealthCheckTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Tests\Service\Management\Health;
+
+use App\Service\Management\Health\AllActiveIpsHaveCorrectPtrHealthCheck;
+use App\Service\Ip\IpAddressService;
+use App\Tests\Case\KernelTestCase;
+use App\Tests\Factory\IpAddressFactory;
+use App\Tests\Factory\QueueFactory;
+use App\Tests\Factory\ServerFactory;
+use PHPUnit\Framework\MockObject\MockObject;
+
+class AllActiveIpsHaveCorrectPtrHealthCheckTest extends KernelTestCase
+{
+    private AllActiveIpsHaveCorrectPtrHealthCheck $healthCheck;
+    private IpAddressService&MockObject $ipAddressService;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        
+        $this->ipAddressService = $this->createMock(IpAddressService::class);
+        
+        $this->healthCheck = new AllActiveIpsHaveCorrectPtrHealthCheck(
+            $this->em,
+            $this->ipAddressService
+        );
+    }
+
+    public function testCheckReturnsTrueWhenNoActiveIpsExist(): void
+    {
+        $this->ipAddressService->expects($this->never())
+            ->method('updateIpPtrValidity');
+
+        $result = $this->healthCheck->check();
+        
+        $this->assertTrue($result);
+        $this->assertEmpty($this->healthCheck->getData());
+    }
+
+    public function testCheckReturnsTrueWhenAllActiveIpsHaveCorrectPtr(): void
+    {
+        $ip = IpAddressFactory::createOne([
+            'is_available' => true,
+            'is_enabled' => true,
+            'is_ptr_forward_valid' => true,
+            'is_ptr_reverse_valid' => true,
+        ]);
+
+        $result = $this->healthCheck->check();
```

Comment:
this test has a side effect via updateIpPtrValidity (calls DNS). Probably use a mock.